dspace-spring-rest/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -89,7 +89,7 @@ public void afterPropertiesSet()  {
 		DSpaceRestRepository<RestModel, ID> repository = utils.getResourceRepository(model);
 		RestModel modelObject = null;
 		try {
-			modelObject = repository.findOne(id);
+			modelObject = repository.findOne(id, projection);
```

Comment:
I agree. Te repository classes implementation should be aware of the requested projection as this could lead in some case to optimized retrieval of information